app/src/containers/ListOfNews/index.js

```diff
@@ -53,7 +54,9 @@ class ListOfNews extends React.Component {
             cors: 'no-cors'
         }).then((response) => response.json())
             .then((responseJson) => {
-                this.setState({ newsList: responseJson })
+                this.setState({ newsList: responseJson
```

Comment:
This value assignment to `newList` should go next line, not the `sort`. Now it looks like `sort` is chained with `setState`